stable/aws-for-fluent-bit/templates/configmap.yaml

```diff
@@ -127,6 +127,29 @@ data:
         {{- end }}
 {{- end }}
 
+{{- if .Values.elasticsearch.enabled }}
+    [OUTPUT]
+        Name            es
+        Match           {{ .Values.elasticsearch.match }}
+        AWS_Region      {{ .Values.elasticsearch.region }}
+        AWS_Auth        {{ .Values.elasticsearch.aws_auth }}
+        {{- if .Values.elasticsearch.host }}
+        host            {{ .Values.elasticsearch.host }}
```

Comment:
Not sure if fluentbit cares about casing, but in the [fluentbit manual](https://docs.fluentbit.io/manual/pipeline/outputs/elasticsearch) this is "Host" not "host"

Reply:
ok